src/components/UploadForms/UploadForm.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
```

Comment:
Remove this file

Reply:
Can we do this later, as part of our technical debt cleanup?

Reply:
yes!